addons/azureSets/data/settings.xml

```diff
@@ -3,6 +3,8 @@
 	<global>
```

Comment:
This is also not your fault, but this file should be deleted. It will be auto-generated (and auto-corrected) on load. It won't help adjusting anything in here anyway, as the `data` folder is exempt from updates. The reason for that is that the `data` folder contains only user-added and user-modified files. In this case we would overwrite all user-settings if we updated this XML.

Instead of adjusting the XML, instead adjust the `defaults` table, which is constructed on line 43, and subsequently passed to the config loader on line 58. Add an entry for `setmode` there:
```lua
defaults.setmode = 'preservetraits'
```

You should delete this file and commit the deletion itself, that way it will be removed from the repository.